src/components/app.tsx

```diff
@@ -10,8 +10,43 @@ import {EncodingPane} from './encoding-pane';
 import {Header} from './header';
 import {ViewPane} from './view-pane';
 
+import {
+  ActionHandler,
+  DatasetAsyncAction,
+  datasetUrlLoad,
+} from '../actions';
+
+
+
+interface DataURL {
+  name: string;
+  url: string;
+};
+export type VoyagerData = DataURL | Object[];
+export type VoyagerConfig = Object;
```

Comment:
looks like tslint disallows empty interfaces. I'll keep in mind changing this to a minimal interface when i actually start working on the config.